Scenes/Characters/player.tscn

```diff
@@ -1,15 +1,62 @@
-[gd_scene load_steps=3 format=3 uid="uid://dkbwkvi8f7mtw"]
+[gd_scene load_steps=6 format=3 uid="uid://dkbwkvi8f7mtw"]
 
 [ext_resource type="Script" path="res://Scripts/Player_Scripts/player.gd" id="1_xnuy8"]
 
-[sub_resource type="CylinderShape3D" id="CylinderShape3D_udcbu"]
+[sub_resource type="CapsuleShape3D" id="CapsuleShape3D_gbg41"]
+radius = 0.4
 height = 1.7
 
+[sub_resource type="Animation" id="Animation_2raob"]
+resource_name = "crouch"
+length = 0.2
+step = 0.0333
+tracks/0/type = "value"
+tracks/0/imported = false
+tracks/0/enabled = true
+tracks/0/path = NodePath("CollisionShape3D:shape:height")
+tracks/0/interp = 1
+tracks/0/loop_wrap = true
+tracks/0/keys = {
+"times": PackedFloat32Array(0, 0.2),
+"transitions": PackedFloat32Array(1, 1),
+"update": 0,
+"values": [1.7, 0.85]
+}
+
+[sub_resource type="Animation" id="Animation_vkb7n"]
+length = 0.001
+tracks/0/type = "value"
+tracks/0/imported = false
+tracks/0/enabled = true
+tracks/0/path = NodePath("CollisionShape3D:shape:height")
+tracks/0/interp = 1
+tracks/0/loop_wrap = true
+tracks/0/keys = {
+"times": PackedFloat32Array(0),
+"transitions": PackedFloat32Array(1),
+"update": 0,
+"values": [1.7]
+}
+
+[sub_resource type="AnimationLibrary" id="AnimationLibrary_lk1yr"]
+_data = {
+"RESET": SubResource("Animation_vkb7n"),
+"crouch": SubResource("Animation_2raob")
+}
+
 [node name="Player" type="CharacterBody3D"]
 script = ExtResource("1_xnuy8")
 
 [node name="CollisionShape3D" type="CollisionShape3D" parent="."]
-shape = SubResource("CylinderShape3D_udcbu")
+shape = SubResource("CapsuleShape3D_gbg41")
 
 [node name="Camera3D" type="Camera3D" parent="."]
-transform = Transform3D(1, 0, 0, 0, 1, 0, 0, 0, 1, 0, 0.677103, 0)
+transform = Transform3D(1, 0, 0, 0, 1, 0, 0, 0, 1, 0, 0.448461, 0)
```

Comment:
Player camera height is just under 0.5m high. Consider changing its heights to better match a person's height.
Also note that I just realised the player placeholder is partially phased through the floor due to it being centred so its not a good indicator of height. Might be worth making a dummy that can be free placed and moved independently.

Height should probably be about 1.5m but this might cause some camera issues with door ways, collider will need updating to match too